README.md

```diff
@@ -19,10 +19,81 @@ Create a `BanyanDBClient` with host, port and a user-specified group and then es
 ```java
 // use `default` group
 client = new BanyanDBClient("127.0.0.1", 17912, "default");
-// establish a connection
+// to send any request, a connection to the server must be estabilished
 client.connect(channel);
 ```
 
+## Schema Management
+
+### Stream
+
+Then we may define a stream with customized configurations,
+
+```java
+// build a stream "sw" with 2 shards and ttl equals to 30 days
+Stream s = new Stream("sw", 2, Duration.ofDays(30));
+s.addTagNameAsEntity("service_id").addTagNameAsEntity("service_instance_id").addTagNameAsEntity("state");
+// TagFamily - data
+TagFamilySpec dataFamily = new TagFamilySpec("data");
+dataFamily.addTagSpec(TagFamilySpec.TagSpec.newBinaryTag("data_binary"));
+s.addTagFamilySpec(dataFamily);
+// TagFamily - searchable
+TagFamilySpec searchableFamily = new TagFamilySpec("searchable");
+searchableFamily.addTagSpec(TagFamilySpec.TagSpec.newStringTag("trace_id"))
```

Comment:
What is the difference between these 2 TagFamily(s)? Especially one says `searchable`, and the other(`binary`) is clearly not searchable. How do these 2 APIs define searchable?

Reply:
> What is the difference between these 2 TagFamily(s)? Especially one says `searchable`, and the other(`binary`) is clearly not searchable. How do these 2 APIs define searchable?

This is only an example. `searchable` and `binary` are names of the tag family

Reply:
Then, please rename them. This is very misleading, because searchable is a very important capability for end users.

Reply:
Use sample names now